code/modules/antagonists/wizard/equipment/spellbook_entries/offensive.dm

```diff
@@ -133,3 +133,24 @@
 	item_path = /obj/item/vibro_weapon/wizard
 	category = "Offensive"
 	cost = 3
+
+/datum/spellbook_entry/item/breakbow
+	name = "Break Bow"
+	desc = "A bladed bow that can be split into two swords which attack simultaneously as well as return to their thrower. Comes with a quiver of unlimited, powerful arrows."
+	item_path = /obj/structure/closet/crate/BreakBow
+	category = "Offensive"
+	cost = 2
+
+/datum/spellbook_entry/magic_arrows
+	name = "Summon Magic Arrows"
+	desc =" summons  magic arrows"
```

Comment:
While it is true, it's not very descriptive. What can the magic arrows do, for example?